drivers/Makefile.dep

```diff
@@ -152,6 +152,11 @@ ifneq (,$(filter fxos8700,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter extend_gpio,$(USEMODULE)))
```

Comment:
I'm sorry if it seems to be a bit nit-picking, but could move this 
```
ifneq (,$(filter extend_gpio,$(USEMODULE)))
```
block before the 
```
ifneq (,$(filter feetech,$(USEMODULE)))
```
block to keep alphabetical order, please. Maybe you placed it there because you initially called it `gpio_extend`.

Reply:
Thanks for catching this. I find the almost-alphabetical order of the makefiles a bit annoying.